doc/endpoints.rst

```diff
@@ -18,6 +18,10 @@ Cluster
 
 (Internal model name: ``Cluster``)
 
+.. versionchanged:: 0.1.0
+
+   ``type`` has been removed in favor of ``cluster_manager``.
+
```

Comment:
@ashcrow Just noticed this: `cluster_manager` should be `container_manager`, right?

Several more cases in the examples below.  You mind if I just fix this without a PR?